lib/ddtrace/contrib/active_job/events/enqueue_at.rb

```diff
@@ -27,6 +27,7 @@ def span_name
           def process(span, event, _id, payload)
             span.name = span_name
             span.resource = payload[:job].class.name
+            span.set_tag(Datadog::Ext::Metadata::TAG_OPERATION, Ext::TAG_OPERATION_ENQUEUE)
```

Comment:
I'm curious: why doesn't enqueue_at get is own tag?

Reply:
Probably a mistake, now it has :)